packages/fxa-payments-server/src/store/actions/api.ts

```diff
@@ -83,14 +83,11 @@ export default {
         };
       },
     } as const),
-  updatePayment: (paymentToken: string, plan: Plan) =>
```

Comment:
Another spot that probably needs replication for the SCA side of payment update, since I added metrics stuff to it.